README.md

```diff
@@ -24,3 +24,13 @@ Or from the package folder:
 NOTE: If you installed it as a GitHub dependency (`npm i --save buildo/react-component`) then import from `/src`:
 
 `import { FlexView } from 'buildo-react-components/src/flex';`
+
+### Versioning guidelines
+A new version of `b-r-c` is published every ~2 weeks with a github release containing a list of all the commits (mostly PR) merged on `master` since the previous one.
```

Comment:
maybe the list should contain only PRs as anything commited directly on master shouldn't be relevant to any project (like fixing an example or the readme)